partcad-cli/src/partcad_cli/click/commands/test.py

```
@@ -12,28 +12,41 @@
 from partcad import logging as logging
 import asyncio
 
+from partcad.test.all import tests as all_tests
 
-async def cli_test_async(ctx, packages, sketch, interface, assembly, scene, object):
+
+async def cli_test_async(ctx, packages, filter_prefix, sketch, interface, assembly, scene, object):
```

Ruff (0.8.2)

`cli_test_async` is too complex (11 > 10)

(C901)